task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -55,10 +57,12 @@ spec:
       FAIL_RESULTS="$(cat fail_result.txt)"
       if [[ -z $FAIL_RESULTS ]]
       then
-        HACBS_TEST_OUTPUT=$(make_result_json -r "SUCCESS" -s 1)
+        note="Task $(context.task.name) completed: Check field `result` for the json check result."
+        HACBS_TEST_OUTPUT=$(make_result_json -r "SUCCESS" -s 1 -t "$note")
       else
         echo "Fail to verify sbom-cyclonedx.json for image $(params.IMAGE_URL) with reason: $FAIL_RESULTS"
-        HACBS_ERROR_OUTPUT=$(make_result_json -r "FAILURE" -f 1)
+        note="Task $(context.task.name) failed: Failed to verify SBOM for image $(params.IMAGE_URL) with reason: $FAIL_RESULTS."
```

Comment:
SR: note="Task $(context.task.name) failed: Failed to verify SBOM for image $(params.IMAGE_URL)."

Reply:
Fixed.